.buildkite/scripts/common/env.sh

```diff
@@ -131,3 +131,17 @@ export TEST_GROUP_TYPE_FUNCTIONAL="Functional Tests"
 
 # tells the gh command what our default repo is
 export GH_REPO=github.com/elastic/kibana
+
+FIPS_ENABLED=false
```

Comment:
Do you think it would make sense to reverse these? i.e  `FIPS_ENABLED` enables `FTR_ENABLE_FIPS_AGENT`, `.buildkite/scripts/steps/checks/verify_fips_enabled.sh`, etc.

Reply:
Ya this makes more sense, thanks. [5e8f1bc](https://github.com/elastic/kibana/pull/183777/commits/5e8f1bcef4ca2f58e5d25f64c1c80d4e3adb3b3a)